### ISSUE TITLE
Support for vlan tagging

### ISSUE THREAD

**Issue by cdmaxson1** [hpe-storage/python-hpedockerplugin#473]
During 3par plugin installation on k8s, the `plugin_container` docker container generates an `invalid iSCSI ip address` error:
```
2019-01-29 18:09:51.654 20 WARNING hpedockerplugin.hpe.hpe_3par_iscsi [-] Found invalid iSCSI IP address(s) in configuration option(s) hpe3par_iscsi_ips or iscsi_ip_address '172.16.16.3, 172.16.16.5, 172.16.16.4, 172.16.16.6.'
2019-01-29 18:09:51.655 20 ERROR hpedockerplugin.hpe.hpe_3par_iscsi [-] At least one valid iSCSI IP address must be set.
```

After initial debugging with HPE team, they believe the issue is related to our use of `vlan tagging` and support must be added to the plugin.

**Comment by prablr79** [comment]
@cdmaxson1 HPE engineering is looking this issue, will be taken as part of 3.1 Release.

**Comment by prablr79** [comment]
Change is validated in our lab environment with VLAN tagging enabled on iscsi ports of 3PAR.

Steps to install this build

1) Copy the .egg file in this mail to a folder (see the attachment)
2) Find the container name of the volume plugin using docker ps -a command like
docker@csosbe03b10:~$ docker ps -a | grep plugin
ebe7245d6dd5        f80c7d1615be                                      "/bin/sh -c ./plug..."   2 weeks ago         Up 4 hours                                                                                                                                   plugin_container

3) Copy the .egg file to the container's site-packages folder using

docker cp  ./python_3parclient-4.2.8-py2.7.egg plugin_container:/usr/lib/python3.6/site-packages/python_3parclient-4.2.7-py3.6.egg

4) docker stop plugin_container
5) docker start plugin_container

Please note: You could substitute the name of the container of volume plugin (here in this case: plugin_container) to the container id itself (which is ebe7245d6dd5 in this example)

6) in another terminal, you could keep the $ docker logs -f plugin_container 2>&1 | tee ~/plugin_initialization.logs


In case if the issue is not resolved, you can send us the above log file generated by the plugin.

**Comment by wdurairaj** [comment]
[python_3parclient-4.2.8-py2.7.zip](https://github.com/hpe-storage/python-hpedockerplugin/files/2860120/python_3parclient-4.2.8-py2.7.zip)
Unzip the above file, and extract the .egg file

**Comment by prablr79** [comment]
Python_3parclient-4.2.9 merged and dist complete

**Comment by sonawane-shashikant** [comment]
Tested and verified
Steps Followed - 
1. Installed hpe 3par docker volume plugin in rhel
2. Configure 3par array having VLAN ISCSI ports
3. Created and mounted volume
4. Verified the mounted volume.

Detailed output for testing

[version.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/3037273/version.txt)
[VLAN output.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/3037274/VLAN.output.txt)

**Comment by wdurairaj** [comment]
Based on the previous comment and the customer's confirmation, I'm closing this defect. Fix available in hpestorage/legacyvolumeplugin:3.1 tag.
